ext/dom/document.c

```diff
@@ -2223,11 +2223,13 @@ PHP_METHOD(domdocument, registerNodeClass)
 		DOM_GET_OBJ(docp, id, xmlDocPtr, intern);
 
 		if (dom_set_doc_classmap(intern->document, basece, ce) == FAILURE) {
```

Comment:
Looking at http://lxr.php.net/xref/PHP_TRUNK/ext/dom/php_dom.c#209 this seems to be simply dead code. We should adjust the function to be void instead.